website/static/js/project.js

```diff
@@ -136,7 +136,7 @@ $(function() {
 
             e.preventDefault();
         } else if ($(e.target).find('#title').val().length > 200) {
-            $('#alert').text('The new component title cannot be more than 200 characters.');
+            $('#alert').text('The new component title cannot be more than 200 characters.'); //This alert never appears...
```

Comment:
Do we need to include an error message if it's not possible for the user to enter a title longer than 200 characters? @sloria